pytest_flask/fixtures.py

```diff
@@ -121,8 +121,14 @@ def _rewrite_server_name(server_name, new_port):
     return sep.join((server_name, new_port))
 
 
-@pytest.fixture(scope='function')
-def live_server(request, app, monkeypatch, pytestconfig):
+def determine_scope(fixture_name, config):
+    if fixture_name is 'live_server':
```

Comment:
We don't need this `if` given `determine_scope` is used only by `live_server`, so let's remove it:

```python
def determine_scope(fixture_name, config):
    return config.getoption('--live-server-scope', 'function')
```